### DISCUSSION TITLE
學生用AI聊天機器人寫作業 教授嘆：難找到代寫鐵證

### DISCUSSION THREAD

**Discussion by ching020721** [1101-GS4719-C/Python-Community-GS4719-C#934] [General]
ChatGPT剛出來的時候就有人說以後可以使用它來寫論文了，起初我還不相信，直到我看到這篇報到，是國外的學生利用ChatGPT寫論文被抓包，我認為可以參考但完全複製黏貼就有點過分了，不僅容易被抓包還很不尊重教授與自己。

https://news.ltn.com.tw/news/world/breakingnews/4166507

**Comment by ruthchen1020** [comment]
如果是我，可能會希望他幫我讀論文吧，幫我先整理大綱，順便幫我把翻譯翻好一點，因為google翻的我看不懂。

**Comment by peggy0329** [comment]
為何這樣沒有鐵證，教授不是已經從可以的段落抓到99%的匹配度了嗎？
不過我也和樓上同學一樣，希望他可以有比谷歌翻譯好的系統...

**Comment by Evan0979** [comment]
學歷越來越不值錢了嗎QQ

**Reply by chia-kai-chang** [reply]
大家的創造力與獨到觀點，相對比較值錢，不要人云亦云，我們可以訓練一下

**Comment by StevenWanglolz** [comment]
直接複製貼上有點誇張了吧，我都是問chatgpt理解完，再拿來參考，絕對不會全部抄襲哈哈哈。不過話說chatgpt真的是寫程式的好幫手!

**Comment by CPCpyt** [comment]
我認為這是對學術的玷汙，
儘管在科技的進步之下這是無可避免，
但身為人類對於技術的使用更有自覺是非常重要的。

**Comment by swerfg03joe** [comment]
剛剛看了一下內文，想知道是哪門課寫論文可以直接透過ChatGPT，原來是哲學課阿，我完全不意外，很多都是只有思辨而不太需要做實驗，去驗證自己的觀點當然容易透過其他工具完成，而他的指導教授也是很神奇，為什麼會懷疑作業是機器人代寫的?機器人代寫的?

**Comment by ru0372572** [comment]
感覺很像唬爛產生器什麼的，用來寫生日祝福其實很好用ㄟ

**Comment by BlahFish1006** [comment]
覺得ChatGPT在學業上絕對是個非常快速又強大的幫手，尤其是需要彙整或接收多方想法的課題，更是省事又完整。
我覺得利用工具來幫助自己節省時間很好，但過度依賴久了感覺人們會逐漸失去自己統整資料和思考的能力，
特別是對現今快速發展的社會，因為這個工作的時間成本實在太大了。

**Comment by harrylee1971** [comment]
chatGPT引領出了一種新的找資料風潮，比起原先的google要人自己去評斷是否他要的資料，AI可以先幫我們判斷一次
但不保證一定是我們要的資料種類就是了

**Comment by a45320614** [comment]
文明與科技在現今發展得非常快速，然而教育模式相對於數百年前卻沒有非常大幅度的成長，我認為目前崛起的AI技術對於教育界是個危機也是個轉機，就像是張家凱老師開設的Python程式設計課程引進了AI技術來輔助評斷學生的學習成效，我個人也非常看好這種教學模式，我認為這可能是未來教育發展及進步的一盞明燈。

**Comment by HsuChen88** [comment]
期末時期ChatGPT忙線中，想要在期末偷懶的人請排隊。

**Comment by 110302550** [comment]
那是一個很好用的程式，但不能這樣使用，有些作業或者是論文不是為了交給老師獲得分數，而是對自己的一個評量。

**Comment by ghost** [comment]
ChatGPT是個很好用的ai，我在複習python 時常問他一些問題，他也總能給出答案，對於學習來說他是很好的學習利器

**Comment by ghost** [comment]
用Ai來寫作業，我認為是這就像和高利貸借錢，雖能解燃眉之急，但在未來還是得花百倍的力氣才能將其補回來，所以使用當慎思，是讓他成為工具，還是成為貸款

**Comment by orianna901001** [comment]
其實我每次使用chatgpt的時候，我都有點擔心這樣這會變成是一種抄襲，但秉持著參考資料的初心，我是不會全複製貼上，希望它單純的成為一個好用的工具

**Comment by llh26** [comment]
我認為ChatGPT適用於你對一件事情不太了解的時候，可以利用ChatGPT快速地去獲取資訊，讓我們對事物有一些初步的認識，再進一步針對我們想瞭解的部分去找尋相關資料。目前，我不認為他提供的資訊可以用複製的方式完完全全被採納，畢竟它在某些領域的回答還是會有錯，雖然未來可能會改善這些問題，但像論文這種學術研究報告，需要有個人觀點的部分，直接複製ChatGPT的內容真的不太優。
